example_remote_control/README.md

```diff
@@ -7,7 +7,6 @@ the LED on another.
 
  1. `s_comms_cp.py` The server application. Run under CPython 3.5+.
  2. `c_comms_tx.py` Transmitting client. Expects a switch to ground on GPIO0.
- Requires `primitives.py` from this repo and `aswitch.py` from
- [this repo](https://github.com/peterhinch/micropython-async).
+ Requires `primitives.py` and `aswitch.py` from this repo [included in this package for convenience].
```

Comment:
I included these file (stripped down version of asyn) for convenience in trying the examples.